_companies/virtuslab-git-repository-organizer-rebase-automation-tool-as-intellij-idea-plugin.markdown

```diff
@@ -37,4 +42,31 @@ copyrights: |
     - Prawa do projektu: MIT License (Open source)
     - Sposób udostępnienia danych; opis i weryfikacja działania projektu na potrzeby pracy licencjackiej: Dane są ogólnodostępne
 ---
-[git machete](https://github.com/VirtusLab/git-machete) is command-line tool for organizing branches in git repository in a tree-like hierarchies, allowing to significantly reduce the effort traditionally associated with maintaining multiple branches in the repository at the same time, especially wrt. rebase. The project main objective would be to provide an IDE plugin with similar functionality. Part of the task is to assess how each CLI feature can be intuitively represented in a GUI plugin.
+
+## Git Machete CLI
+
+[git machete](https://github.com/VirtusLab/git-machete#git-machete) 
+is command-line tool for organizing branches in git repository in a tree-like hierarchies, 
+allowing to significantly reduce the effort traditionally associated 
+with maintaining multiple branches in the repository at the same time, especially wrt. rebase. 
+The project main objective would be to provide an IDE plugin with similar functionality. 
+Part of the task is to assess how each CLI feature can be intuitively represented in a GUI plugin.
+
+## Git Machete IntelliJ Plugin
+
+This plugin adds an extra tab in the Git tool window that gives the instant answer to the questions: 
+- what are the branches in this repository? 
+- what needs be merged/rebased/pushed/pulled?
+
+
+![Git Machete Tab](https://raw.githubusercontent.com/VirtusLab/git-machete-intellij-plugin/master/docs/pz-ss.png)
```

Comment:
Here comes our ss. Let's make it available from our repo first.

@micpiotrowski can you add one under `docs/pz-ss.png` in the scope of https://github.com/VirtusLab/git-machete-intellij-plugin/pull/452?

Reply:
![Screenshot from 2020-08-11 11-19-11](https://user-images.githubusercontent.com/19799111/89880703-d31a2380-dbc4-11ea-8d7a-6d9c3ed52aee.png)

Reply:
As we agreed via Slack, we can use `sample_workflow.gif` here. It will be available as `https://raw.githubusercontent.com/VirtusLab/git-machete-intellij-plugin/develop/docs/sample_workflow.gif` after VirtusLab/git-machete-intellij-plugin#452 merge

Reply:
Or even as `https://raw.githubusercontent.com/VirtusLab/git-machete-intellij-plugin/doc/readme/docs/sample_workflow.gif` right now (e.g. for tests)

Reply:
Ok, as agreed by SLack later, new sample gif has been created. For now, it's available here: https://raw.githubusercontent.com/VirtusLab/git-machete-intellij-plugin/doc/readme/docs/pz_sample_workflow.gif or shorter version here: https://raw.githubusercontent.com/VirtusLab/git-machete-intellij-plugin/doc/readme/docs/pz_sample_workflow_shorter.gif

Reply:
see https://github.com/VirtusLab/git-machete-intellij-plugin/pull/452#issuecomment-672783182